android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ConfigService.kt

```diff
@@ -100,8 +100,12 @@ interface ConfigService {
       val paramExpression = sp.expression
       val expressionValue =
         when (paramName) {
-          ConfigurationRegistry.ORGANIZATION -> authenticatedUserInfo?.organization
-          ConfigurationRegistry.PUBLISHER -> authenticatedUserInfo?.questionnairePublisher
+          // TODO: Does not support multi organization yet
```

Comment:
Is the todo here correct? If so we need to create an issue to track that

Reply:
Yeah, created an issue to track this https://github.com/opensrp/fhircore/issues/1550